src/makeRobotAccountant.js

```diff
@@ -6,7 +6,21 @@
  */
 
 function makeRobotAccountant() {
-  // write code here
+  let count = 0;
+
+  const getSum = (firstVar) => {
+    count++;
+
+    const typeCondition = count < 3 || count % 2 === 1;
+
+    if (typeCondition) {
+      return (secondVar) => firstVar + secondVar;
```

Comment:
what is first var and second var?

Reply:
like primal and secondary argument in two calls, will rename them 👓